src/color/mfcamerareader.cpp

```diff
@@ -539,14 +539,165 @@ void CMFCameraReader::Stop()
     }
 }
 
+k4a_result_t CMFCameraReader::GetCameraControlCapabilities(const k4a_color_control_command_t command,
+                                                           color_control_cap_t &capabilities)
```

Comment:
>& [](start = 79, length = 1)

avoid non-const lvalue references if possible - it makes it easier to tell that the thing you're passing in is being manipulated

Reply:
I will use pointer.